### PR TITLE
adding tf68 keymap, my first pull request. expecting there's lots of bad practice, sorry

### PR DESCRIPTION
**Pull request by Rossman360**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/tragicforce68/keymaps/rossman360/keymap.c [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+      case COPYCLOSE:
+   if (record->event.pressed) {
+          SEND_STRING(SS_LCTRL(SS_TAP(X_C)));
+          SEND_STRING(SS_LCTRL(SS_TAP(X_W)));
+      }
+      break;
+    };  
+  return true;
+  
+};
```
Comment (ridingqwerty) [inline keyboards/keebio/tragicforce68/keymaps/rossman360/keymap.c]: The indentation here could use some love, but functionally this looks ok.  My only other suggestion would be to convert all these multi-line SEND_STRINGs into a single line like so, unless you are doing this out of a desire for readability:

```
SEND_STRING(SS_TAP(X_PGDOWN));
SEND_STRING(SS_TAP(X_ENTER));
SEND_STRING(SS_TAP(X_ENTER));
SEND_STRING(SS_TAP(X_PGDOWN));
```

Could be condensed to:

```
SEND_STRING(SS_TAP(X_PGDOWN) SS_TAP(X_ENTER) SS_TAP(X_ENTER) SS_TAP(X_PGDOWN));
```

**Comment by noroadsleft** [comment]
For future reference, we recommend against committing to your `master` branch as you've done here, because pull requests from modified `master` branches can make it more difficult to keep your QMK fork updated. It is highly recommended for QMK development – regardless of what is being done or where – to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing.

There are instructions on how to keep your fork updated here:

[**Best Practices: Your Fork's Master: Update Often, Commit Never**](https://docs.qmk.fm/#/newbs_git_using_your_master_branch)

If you need any help with this just ask.

**You do not need to close this pull request at this time.** Until your pull request is merged, please continue to commit relevant changes to your `master` branch.

File: keyboards/keebio/tragicforce68/keymaps/rossman360/keymap.c (hunk 1 above)
Comment (Rossman360) [reply keyboards/keebio/tragicforce68/keymaps/rossman360/keymap.c]: love it.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**